classes/requests/class-kp-requests-patch.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Base class for all PATCH requests.
+ *
+ * @package WC_Klarna_Payments/Classes/Request
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ *  The main class for PATCH requests.
+ */
+abstract class KP_Requests_Patch extends KP_Requests {
+
+	/**
+	 * Qliro_One_Request_Post constructor.
```

Comment:
Fix the comment here. Wrong class name

Reply:
Seems like we have the same in the post class as well, so can you fix that too while you are doing this one?